openhtf/plugs/user_input.py

```diff
@@ -58,34 +58,31 @@ class PromptUnansweredError(Exception):
 
 
 class ConsolePrompt(threading.Thread):
-  """Thread that displays a prompt to the console and waits for a response.
+  """Thread that displays a prompt to the console and waits for a response."""
 
-  Args:
-    prompt_id: The prompt manager's id associated with this prompt.
-  """
   def __init__(self, message, callback):
     super(ConsolePrompt, self).__init__()
     self.daemon = True
     self._message = message
     self._callback = callback
     self._stopped = False
+    self._answered = False
```

Comment:
Using this to distinguish between a response that came from elsewhere and a response that came asynchronously. (It was confusing to get the message "Nevermind; prompt was answered from elsewhere." after answering the prompt asynchronously in the console.)